frontend/src/metabase/lib/uploads.ts

```diff
@@ -0,0 +1,9 @@
+import { FileUpload } from "metabase-types/store/upload";
+
+export const uploadInProgress = (upload: FileUpload) =>
```

Comment:
nit: in most places, we prefer using `isUploadInProgress` over `uploadInProgress` for predicates